green/test/test_loader.py

```diff
@@ -264,6 +264,16 @@ def test_method2(self):
                          set(['test_method1', 'test_method2']))
 
 
+    def test_nose_disabled_attribute(self):
+        "Tests disabled by nose generators dont get loaded"
+        class HasDisabled(unittest.TestCase):
+            def test_method(self):
+                pass
+
+        HasDisabled.test_method.__test__ = False
```

Comment:
Thanks for the patch!  Unfortunately, the test fails on all of the builders except Python 3.4 on Windows.  From what I've looked at so far, it's this line in the test itself that crashes.

```python
$ python
Python 2.7.6 (default, Sep  9 2014, 15:04:36)
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.39)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> class A:
...     def b(self):
...             pass
...
>>> A.b.__test__ = False
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'instancemethod' object has no attribute '__test__'
```

I'm not sure what the workaround is.  I've created attributes on bare functions before, but not on class methods.